### ISSUE TITLE
migration guide v4 -> v5

### ISSUE THREAD

**Issue by oprogramador** [hughsk/flat#91]
Could you publish a migration guide explaining how to correctly upgrade `flat` from v4 into v5?

At https://github.com/hughsk/flat/releases I see only an announcement of `5.0.0` with no info about breaking changes but I'd like to ensure.

**Comment by christianbundy** [comment]
Same, I've skimmed the changes but can't find the breaking change.

https://github.com/hughsk/flat/compare/4.1.0...5.0.0

**Comment by yvele** [comment]
Yep and please write a changelog (in the GitHub release notes https://github.com/hughsk/flat/releases/tag/5.0.0)

What exactly is breaking in 5.0.0?

**Comment by ossman11** [comment]
When looking at the https://github.com/hughsk/flat/compare/4.1.0...5.0.0 compare the main point is probably the [```const```](https://github.com/hughsk/flat/compare/4.1.0...5.0.0#diff-168726dbe96b3ce427e7fedce31bb0bcR22-R26) usage instead of the ```var``` which limits the ```Node``` versions or ```browsers``` supported.

**Comment by jonkoops** [comment]
Version 6 will be released soon and will include a proper migration guide. There should be no breaking changes for version 5 besides usage of newer JavaScript syntax.
